tests/cli/main.cpp

```diff
@@ -1130,10 +1130,10 @@ BOOST_FIXTURE_TEST_CASE( cli_sign_message, cli_fixture )
 
    // change account, verify failure
    msg.meta.account = "dan";
```

Comment:
Perhaps it's better to also try to verify with an existing account, E.G. committee-account.

Reply:
OK, how's that?